libs/react-helpers/src/lib/grid/vol-cell.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import type { ICellRendererParams } from 'ag-grid-community';
+import { PriceCell } from './price-cell';
+
+export interface VolProps {
+  value: number | bigint | null | undefined;
+  relativeValue?: number;
+  type: 'bid' | 'ask';
+}
+export interface IVolCellProps extends ICellRendererParams {
+  value: number | bigint | null | undefined;
+  valueFormatted: Omit<VolProps, 'value'>;
+}
+
+export const BID_COLOR = 'darkgreen';
```

Comment:
chore (non-blocking): Can we use tailwind theme colors here?

Reply:
@campbellssource
Bar background colors are not present on styleguide colors palette. We need to extend palette or amend colors.

Reply:
https://github.com/vegaprotocol/frontend-monorepo/issues/297